### ISSUE TITLE
Type UX Suggestion

### ISSUE THREAD

**Issue by mmport80** [isovector/algebra-driven-design#3]
```
∀ (w :: Int) (h :: Int) (t1 :: Tile (a -> b))
      (t2 :: Tile a).
  rasterize' w h (t1 <*> t2) =
    rasterize' w h t1 <*> rasterize' w h t2
```

The `Tile (a -> b)` threw me for a few hours.

Due to reading on an ebook, and expecting `t1` to be `Tile a`.

Not a big deal, but perhaps there's a way to make the nomenclature clearer.

Nice book!

**Comment by isovector** [comment]
Maybe `tab` and `ta` would be better names than `t1` and `t2`?

**Comment by mmport80** [comment]
yep maybe it will improve things. (maybe they problem was the ebook layout - good lesson for me to read my ipad in portrait mode..)
